src/main/kotlin/com/exactpro/th2/common/schema/grpc/configuration/GrpcRawFilterStrategy.kt

```diff
@@ -16,14 +16,53 @@
 package com.exactpro.th2.common.schema.grpc.configuration
 
 import com.exactpro.th2.common.schema.configuration.Configuration
+import com.exactpro.th2.common.schema.message.configuration.DESERIALIZER_MULTI_MAP_FIELD_FILTER
 import com.exactpro.th2.common.schema.message.configuration.FieldFilterConfiguration
 import com.exactpro.th2.common.schema.message.configuration.RouterFilter
+import com.exactpro.th2.common.util.emptyMultiMap
+import com.fasterxml.jackson.annotation.JsonGetter
 import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.core.JsonParseException
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.JsonDeserializer
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.module.SimpleModule
+import org.apache.commons.collections4.MultiMapUtils
+import org.apache.commons.collections4.MultiValuedMap
 
 data class GrpcRawFilterStrategy(var filters: List<GrpcRouterFilterConfiguration?> = emptyList())
 
 data class GrpcRouterFilterConfiguration(
     @JsonProperty(required = true) var endpoint: String,
-    override var metadata: Map<String, FieldFilterConfiguration> = emptyMap(),
-    override var message: Map<String, FieldFilterConfiguration> = emptyMap()
-) : RouterFilter, Configuration()
\ No newline at end of file
+    override var metadata: MultiValuedMap<String, FieldFilterConfiguration> = emptyMultiMap(),
+    override var message: MultiValuedMap<String, FieldFilterConfiguration> = emptyMultiMap()
+) : RouterFilter, Configuration() {
+
+    @JsonGetter("metadata")
+    fun getJsonMetadata(): Collection<FieldFilterConfiguration> = metadata.values()
+
+    @JsonGetter("message")
+    fun getJsonMessage(): Collection<FieldFilterConfiguration> = message.values()
+
+}
+
+class GrpcRouterFilterConfigurationDeserializer : JsonDeserializer<GrpcRouterFilterConfiguration>() {
```

Comment:
Maybe we can create a single deserializer for MultiValuedMap<String, FieldFilterConfiguration> and apply it to the fields using `@JsonDeserializer` annotation?
We won't need to register any modules in ObjectMapper. Also, we won't need to check if the key is present (the deserializer won't invoked at all)